parts/k8s/kubernetesparameters.t

```diff
@@ -0,0 +1,10 @@
+{
+  "parameters": {
```

Comment:
it's slightly confusing that we have both `kubernetesparams` and `kubernetesparameters` files... Maybe this one can be `parametersbase` or something

Reply:
I would like to park this as I feel we'd need to discuss how we want to go forward with aks-engine as a tool. (Is it more of an ARM utility or more of a Kubernetes provisioner on Azure?)

Reply:
I agree this is confusing. How about "arm_template_parameters.t"?

I think changing this filename (now, or in the future) is trivial, so we don't have to pre-anticipate everything to determine whether or not we change the name. Let's just change it now to add a slight aid to codebase maintenance (if folks agree that a suggestion like mine accomplishes that).

Reply:
changing it to `armparamters.t`